kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -57,49 +56,106 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
         >>>     credentials=("username", "password")
         >>> )
         >>> data = data_set.load()
+
+    ``APIDataSet`` can also be used to save output on a remote server using
+    HTTP(S) methods.
+
+        >>> example_table = '{"col1":["val1", "val2"], "col2":["val3", "val4"]}'
+
+    Here we initialise our APIDataSet with the correct parameters to make requests
+    towards the configured remote server.
+
+        >>> data_set = APIDataSet(
+                method = "POST"
+                url = "url_of_remote_server",
+                save_args = {"chunk_size":1}
+        )
+    On initialisation, we can specify all the necessary parameters in the save args
+    dictionary. The default HTTP(S) method is POST but PUT is also supported.
+    Two important parameters to keep in mind are timeout and chunk_size. ``Timeout``
```

Comment:
```suggestion
    Two important parameters to keep in mind are `timeout` and `chunk_size`.    
```

Does this render into something special? There are many capitalised version of the parameters. If it's a variable then use `` `` to quote it.